app/src/lib/stores/app-store.ts

```diff
@@ -351,6 +351,7 @@ const pullRequestFileListConfigKey: string = 'pull-request-files-width'
 
 const askToMoveToApplicationsFolderDefault: boolean = true
 const confirmRepoRemovalDefault: boolean = true
+const showCommitLengthWarningDefault: boolean = true
```

Comment:
```suggestion
const showCommitLengthWarningDefault: boolean = false
```

Going to set the default to our desired default which is false, but in another commit going to add a statement to set back to true in local storage so that current users will maintain their current state. This will give us the ability to simply remove that if statement when we want the desired default to flip for new users but not existing.